symfony/src/Entity/PatientStatus.php

```diff
@@ -1,134 +0,0 @@
-<?php
```

Comment:
Why is this removed?

Reply:
I must have picked up the delete when trying to clean up after running `doctrine:mapping:import`. 🐼 Fixed in 
9e6d25f